wallet/craco.config.js

```diff
@@ -8,4 +10,12 @@ module.exports = {
       ],
     },
   },
+  webpack: {
+    plugins: [
+      // include process.env.STAGE provided during build at runtime
+      new DefinePlugin({
+        'process.env.STAGE': JSON.stringify(process.env.STAGE ?? 'prod')
+      })
```

Comment:
Just note, this doesn't actually update `process.env` (it just replaces the explicit use of `process.env.STAGE` as above)
- it is possible to include it in `process.env` via a callback in CRA (it's just a bit hacky though)